### PR TITLE
Sdap 1006 cutandrun feedback changes and dna binding wf output variable change

### PR DESCRIPTION
**Pull request by robert-player**

### CODE REVIEW

**Comment by michael-kotliar** [comment]
Also, I would update all of the occurrences of `macs2_called_peaks` in the `conformance_tests.yaml` file and make sure that neither of the tests fails by running `run_test_docker.sh`

File: workflows/trim-atacseq-pe.cwl [hunk 1]
```diff
@@ -20,6 +20,7 @@ requirements:
   genome_indices: "genome-indices.cwl"
   control_file: "trim-atacseq-pe.cwl"
 
+
```
Comment (scrowley-Datirium) [inline workflows/trim-atacseq-pe.cwl:23]: remove extra line on 23 so minor update doesn't occur

File: workflows/trim-atacseq-se.cwl [hunk 2]
```diff
@@ -19,6 +19,7 @@ requirements:
   genome_indices: "genome-indices.cwl"
   control_file: "trim-atacseq-se.cwl"
 
+
```
Comment (scrowley-Datirium) [inline workflows/trim-atacseq-se.cwl:22]: remove extra line so minor update doesn't occur

File: workflows/trim-chipseq-se.cwl [hunk 3]
```diff
@@ -20,6 +20,7 @@ requirements:
   genome_indices: "genome-indices.cwl"
   control_file: "trim-chipseq-se.cwl"
 
+
```
Comment (scrowley-Datirium) [inline workflows/trim-chipseq-se.cwl:23]: remove extra line so minor update doesn't occur

File: workflows/diffbind.cwl [hunk 4]
```diff
@@ -1418,7 +1418,7 @@ doc: |
   sequence read datasets. It is designed to work with multiple peak sets simultaneously,
   representing different ChIP or CUT&RUN/Tag experiments (antibodies, transcription factor
   and/or histone marks, experimental conditions, replicates) as well as managing the results
-  of multiple peak callers.
+  of multiple peak callers.
```
Comment (scrowley-Datirium) [inline workflows/diffbind.cwl:1421]: remove extra space on line 1421

File: workflows/manorm-se.cwl [hunk 5]
```diff
@@ -17,7 +17,7 @@ requirements:
     - "chipseq-se.cwl"
     - "trim-chipseq-se.cwl"
     - "trim-atacseq-se.cwl"
-
+
```
Comment (scrowley-Datirium) [inline workflows/manorm-se.cwl:20]: remove new line

**Review by scrowley-Datirium — APPROVED** [review]
LGTM

**Pull request merged**